compute_endpoint/globus_compute_endpoint/endpoint/config/user_config_template.yaml

```diff
@@ -0,0 +1,43 @@
+# This is the default user-template provided with newly-configured Multi-Tenant
+# endpoints.  User endpoints generate a user-endpoint-specific configuration by
+# processing this YAML file as a Jinja template against user-provided
+# variables -- please modify this template to suit your site's requirements.
+#
+# For more information, please see the `user_endpoint_config` in Globus Compute
+# SDK's Executor.
+#
+# Common options site-administrators may want to set here include:
+#  - address
+#  - provider (e.g., SlurmProvider, TorqueProvider, CobaltProvider, etc.)
+#  - account
+#  - scheduler_options
+#  - walltime
```

Comment:
Some commented out in-line versions of this would be really helpful.  Even for the simple items like walltime, the type of value (int, float, str) and indentation levels are not obvious to a new admin of compute.

Reply:
Per discussion, have put in a link to the example configurations.